src/core/sagas/collection.ts

```diff
@@ -0,0 +1,30 @@
+import * as effects from 'redux-saga/effects';
+import FluxCapacitor from '../../flux-capacitor';
+import Actions from '../actions';
+import Adapter from '../adapters/search';
+import Selectors from '../selectors';
+import Store from '../store';
```

Comment:
it needs the default export the way it is, otherwise typescript is unable to link the types correctly and will fail to transpile

also the correct form would just be `import '../store';`